src/commands/scanner/run.ts

```diff
@@ -157,6 +157,7 @@ export default class Run extends ScannerCommand {
       // If we were given a file, we should write the output to that file.
       try {
         fs.writeFileSync(this.flags.outfile, output);
+        this.ux.log(messages.getMessage('output.writtenToOutFile', [this.flags.outfile]));
```

Comment:
As a heads up, this will probably cause some tests to fail.

Reply:
I fixed the test as well.